xbmc/addons/Scraper.cpp

```diff
@@ -618,8 +618,6 @@ CMusicAlbumInfo FromFileItem<CMusicAlbumInfo>(const CFileItem &item)
     sAlbumName = sTitle;
 
   std::string sYear = item.GetProperty("album.year").asString();
```

Comment:
sYear isn't used in this routine now, delete

Reply:
good catch, will remove it.